src/router/Router.tsx

```diff
@@ -34,35 +33,31 @@ function Router() {
     event.preventDefault();
     setValue(newValue);
   };
-  
+
   return (
     <>
-      <AppBar position="static">
-        <Tabs
-          variant="fullWidth"
-          value={value}
-          onChange={handleChange}
-        >
-          <LinkTab label="Home" to={Routes.home} />
-          <LinkTab label="About Us" to={Routes.About}/>
+      <AppBar position="static" style={{ backgroundColor: "#FF8C42" }}>
+        <Tabs variant="fullWidth" value={value} onChange={handleChange}>
+          <div id="my-panel">
```

Comment:
What's this div for?

Reply:
I was playing around with using *hover to try and get the color to change for the text when someone hovers over the navbar elements. Still a work in progress.